classes/module/Module.php

```diff
@@ -1045,6 +1060,125 @@ public function editExceptions($id_hook, $excepts)
         return $result;
     }
 
+    /**
+     * Install a tab according to its defined structure
+     *
+     * @param array $tab_info The structure of the tab.
+     *
+     * @return bool   true if the tab was installed successfully, false otherwise
+     *                (and a descriptive error will be added to $_errors)
+     */
+    private function installTab($tab_info)
```

Comment:
I seem to remember it is the case. The name can be changed to something more private, I was just trying to make the tab-management code more self-contained.